demos/appyx-interactions/android/src/main/kotlin/com/bumble/appyx/interactions/bottomnav/Icons.kt

```diff
@@ -0,0 +1,69 @@
+package com.bumble.appyx.interactions.bottomnav
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.materialIcon
+import androidx.compose.material.icons.materialPath
+import androidx.compose.ui.graphics.vector.ImageVector
+
+internal val Icons.Filled.GridViewCustom: ImageVector
+    // Based on Icons.Filled.GridView
+    get() {
+        if (_gridView != null) {
+            return _gridView!!
+        }
+        _gridView = materialIcon(name = "Outlined.GridView") {
+            materialPath {
+                moveTo(3.0f, 3.0f)
+                verticalLineToRelative(8.0f)
+                horizontalLineToRelative(8.0f)
+                verticalLineTo(3.0f)
+                horizontalLineTo(3.0f)
+                close()
+//                moveTo(9.0f, 9.0f)
```

Comment:
delete?

Reply:
This is a modified icon I hacked together by a bit of trial and error. I thought it was useful to see where it differs from the original in case further modification is needed.